spearhead/style-rtl.css

```diff
@@ -363,6 +364,16 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
 	color: var(--primary-nav--color-link-hover);
 }
 
+.has-title-and-tagline .site-logo {
+	flex-basis: 100%;
+	margin-bottom: var(--global--spacing-vertical);
+	text-align: center;
+}
+
+.primary-navigation-open .has-title-and-tagline .site-logo {
+	display: none;
```

Comment:
This makes the open animation quite janky, but the alternative is to have the header be really large which looks odd...

Reply:
Seems fine for now — we can always spend some time revising the animation post-launch if we decide to.